api-catalog-services/src/main/resources/application.yml

```diff
@@ -35,14 +35,14 @@ mfaas:
         fetchRegistry: true
         registerWithEureka: true
         region: default
-        nonSecurePortEnabled: true
-        securePortEnabled: false
+        nonSecurePortEnabled: false
+        securePortEnabled: true
         eurekaUserName: ${environment.eurekaUserId}
         eurekaUserPassword: ${environment.eurekaPassword}
         endpoints:
-            statusPage: ${mfaas.server.contextPath}/application/info
-            healthPage: ${mfaas.server.contextPath}/application/health
-            homePage: /ui/v1/apicatalog
+            homePage: https://${mfaas.service.hostname}:${mfaas.server.port}${mfaas.server.contextPath}
```

Comment:
I would use '${mfaas.service.scheme}://' instead of https://'
Consider also changing mfaas to apiml.

Reply:
I told Andrea not to change mfaas to apiml in this PR.

Reply:
Done :) Regarding the change to `apiml`, this `application.yml` needs to be refactored but probably it will take more efforts, so I think it makes sense to have it as a separated story.